### ISSUE TITLE
addMarkers debugging

### ISSUE THREAD

**Issue by sdellachiesa** [sdellachiesa/KST#3]
Help :) 
I try to explain :)
Filter markers by elevation it works just fine each time I use the slider.
However, the contents of the Marker properties (image url, wiki url, name, etc..) are not consistent.

Basically each time I use the slider the markers which are filtered change name, img url, wiki, url, etc...
The input data table has 162 records, and if with the slider I set an elevation that filter 50 stations the markers will use the properties of the last 50 records of the initial data table. Is not using the records filtered by that elevation.

I read the help of the reactive and observe function, tried to use, "Isolate, reactiveValues..." to pass to the observe function the "reactive filtered data" of course I was not successful.. 
Likely I do not understand how should I use the reactive and observe functions.

**Comment by GiulioGenova** [comment]
I tried and seems to work just fine.
No elevation filter
![immagine](https://user-images.githubusercontent.com/39799547/71478249-e374be00-27ee-11ea-9775-bec066d2b3ff.png)
Elevation filter
![immagine](https://user-images.githubusercontent.com/39799547/71478298-2a62b380-27ef-11ea-9322-03869b6d065d.png)
Could you double check?

**Comment by sdellachiesa** [comment]
I see... the popup content is actually correct but the colours of the icons are not.
I was wrongly filtering the icon´s url in the wrong place and I was assigning the colours to the icons OUTSIDE the observe function.


I solved it but assigning the url´s at the begin in the data frame and assign the icon inside the observe Function :)

Thanks for your feedback, the problem was in front my nose, I saw it just because of your comment.
